api/database/helpers/ApiEnums.php

```diff
@@ -105,4 +105,6 @@ public static function salaryRanges() : array
     const CANDIDATE_STATUS_EXPIRED = 'EXPIRED';
     const CANDIDATE_STATUS_PLACED_CASUAL = 'PLACED_CASUAL';
 
+    const POOL_STATUS_NOT_TAKING_APPLICATIONS = 'NOT_TAKING_APPLICATIONS';
+    const POOL_STATUS_TAKING_APPLICATIONS = 'TAKING_APPLICATIONS';
```

Comment:
Should this be used in other places where `TAKING_APPLICATIONS` and `NOT_TAKING_APPLICATIONS` is used?

Just these, I think:
`api/database/factories/PoolFactory.php`
`api/database/seeders/PoolSeederUat`